mocha/fixtures/streams/AuthorBar.fixtures.js

```diff
@@ -0,0 +1,19 @@
+import _ from "lodash/core"
+
+const getPropsData = function (args = {}) {
```

Comment:
I kind of feel the fixtures should be located where the components and tests are, so in `socialhome/streams/app/tests/fixtures/`. Now they're isolated from both the code and the tests, which seems weird.